snorkel/labeling/model/label_model.py

```diff
@@ -637,7 +731,7 @@ def save(self, destination: str, **kwargs: Any) -> None:
 
         Example
         -------
-        >>> label_model.save('./saved_label_model')
+        >>> label_model.save('./saved_label_model')  # doctest: +SKIP
```

Comment:
Looks like the base is incorrect. A lot of these doctest fixes are already in redux, e.g. https://github.com/HazyResearch/snorkel/blob/redux/snorkel/labeling/model/label_model.py#L643

Reply:
This was the manual fix to make sure the branch doesn’t break. Rebase says it’s updated, I could git pull but that’ll push all the other commits in her too

Reply:
can you rebase or cherry-pick instead of merge? you can also set the pull default to rebase https://coderwall.com/p/tnoiug/rebase-by-default-when-doing-git-pull